src/checks/noDependencyProblems/noUnusedDependencies.ts

```diff
@@ -58,7 +59,7 @@ export default async function noUnusedAndMissingDependencies() {
 
   try {
     await execaCommand(
-      `${preflightBinPath}/depcheck --ignores="${ignoredPackagePatterns}"`,
+      `${preflightBinPath}/depcheck --ignores="${ignoredPackagePatterns}" --ignores-patterns="${ignoredFilePatterns}"`,
```

Comment:
can you link to the docs on this?

Reply:
[Depcheck docs](https://github.com/depcheck/depcheck#usage)

Reply:
copying here for discussion:

```
--ignores: A comma separated array containing package names to ignore. It can be glob expressions. Example, --ignores="eslint,babel-*".

--ignore-dirs: DEPRECATED, use ignore-patterns instead. A comma separated array containing directory names to ignore. Example, --ignore-dirs=dist,coverage.

--ignore-path: Path to a file with patterns describing files to ignore. Files must match the .gitignore [spec](http://git-scm.com/docs/gitignore). Example, --ignore-path=.eslintignore.

--ignore-patterns: Comma separated patterns describing files to ignore. Patterns must match the .gitignore [spec](http://git-scm.com/docs/gitignore). Example, --ignore-patterns=build/Release,dist,coverage,*.log.
```

Reply:
Using the `-ignore-patterns` option is the best workaround for me because it allows us to ignore files matching a specific pattern. I think this is useful because the file is not a package and could have different paths in other projects.